pypi_formula_mappings.json

```diff
@@ -844,6 +844,24 @@
   "pyspelling": {
     "exclude_packages": ["lxml", "markdown", "PyYAML", "six"]
   },
+  "python@3.7": {
+    "extra_packages": ["setuptools", "pip", "wheel"]
+  },
+  "python@3.8": {
+    "extra_packages": ["setuptools", "pip", "wheel"]
+  },
+  "python@3.9": {
+    "extra_packages": ["setuptools", "pip", "wheel"]
+  },
+  "python@3.10": {
+    "extra_packages": ["setuptools", "pip", "wheel"]
+  },
+  "python@3.11": {
+    "extra_packages": ["setuptools", "pip", "wheel"]
+  },
+  "python@3.12": {
+    "extra_packages": ["setuptools", "pip", "wheel"]
+  },
```

Comment:
you also need `flit-core`

Reply:
We can also leave out 3.7 since it's end of life and disabled.

Reply:
👍

Reply:
> you also need `flit-core`

I saw that in some of the versions but not others; do you know where it came from? AFAICT Python itself doesn't use `flit-core` anywhere in its build.

Reply:
@woodruffw there is come context in this thread:

https://github.com/Homebrew/homebrew-core/pull/127641
https://github.com/pypa/wheel/pull/501